util/Makefile

```diff
@@ -7,8 +7,9 @@ CXX ?= g++
 
 CXXFLAGS = $(shell pkg-config --cflags rocksdb)
 LIBS = $(shell pkg-config --static --libs rocksdb)
+INC = $(shell dirname $(abspath $(dir $$PWD)))
```

Comment:
```suggestion
INC = $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
```

Reply:
PWD depends on where you build from. :)

Reply:
@MaisenbacherD Thanks, PTAL. It seems this Makefile would only be used inside the `util` directory to compile `zenfs.cc` manually. The CMake script is not affected by this Makefile file.